xml/security_ldap_install.xml

```diff
@@ -429,6 +429,30 @@ binddn = cn=Directory Manager
      </para>
     </callout>
    </calloutlist>
-  </example>
+  </example>  
+       <warning>
+       <title>New negation feature in sudoers.ldap</title>
+       <para>
+         If you are using sudoers.ldap, instead of the traditional 
+         <command>sudo</command> command, one of its limitations is 
+         negation does not work as you might expect in the 
```

Comment:
"as you might expect" seems like odd language choice for docs. I think it should be "limitations is negation in sudo rules was not available".